tests/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects.Tests/AppHostTests.cs

```diff
@@ -0,0 +1,16 @@
+using CommunityToolkit.Aspire.Testing;
+using Aspire.Components.Common.Tests;
+using FluentAssertions;
+
+namespace CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects.Tests;
+
+[RequiresDocker]
+public class AppHostTests(AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_SqlDatabaseProjects_AppHost> fixture) : IClassFixture<AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_SqlDatabaseProjects_AppHost>>
```

Comment:
Would it make sense to have a test which queries the database and verifies that the table exists? It's possible to access the connection string of the SQL database (see [these docs on testing](https://learn.microsoft.com/dotnet/aspire/testing/accessing-resources?pivots=mstest#access-other-resources)) and then provide that to a `SqlConnection` instance to then build out a query.